### PR TITLE
2.8.2 release

### PR DESCRIPTION
**Pull request by The-Compiler**
Wheee, my first release! :tada:

**Please don't merge this, just comment instead - I'd like to merge it myself when I have time to do the rest of the release (probably tomorrow)**

2.8.2 is uploaded to devpi: `devpi use https://devpi.net/The-Compiler/dev && devpi test pytest==2.8.2`

Unfortunately I ran into a lot of issues during testing (see [devpi](https://devpi.net/The-Compiler/dev/pytest/2.8.2/+toxresults/pytest-2.8.2.tar.gz)):

- plugins_index.py: Wrong version number? (#1118)
- Release annoucements missing for 2.7.2/2.8.0/2.8.1(#1117)
- test_pdb.py hangs with tox -e py27-pexpect on Archlinux when run with devpi (#1116)
- Many failures with tox -e py34 on Windows (#1115)
- tox -e py35-xdist (sometimes) fails on OS X (test_cache_show) (#1114)
- devpi generates funny hostnames with OS X [devpi issue 276](https://bitbucket.org/hpk42/devpi/issues/276/weird-hostname-on-os-x)

I think none of those should block 2.8.2 though - the test failures smell like they have been introduced in 2.8.0 or even earlier.

Because of some incomplete test machine setups (I just recycled what I use for qutebrowser :wink:) I was unable to test some configurations, so I'd be glad if someone else could.

### CODE REVIEW

File: doc/en/announce/release-2.8.2.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    http://pytest.org
+
+As usual, you can upgrade from pypi via::
+
+    pip install -U pytest
+
+Thanks to all who contributed to this release, among them:
+
+    Bruno Oliveira
```
Comment (nicoddemus) [inline doc/en/announce/release-2.8.2.rst]: Out of curiosity, how did you obtain this list? From the changelog?

**Comment by nicoddemus** [comment]
> Wheee, my first release! :tada:

Thanks for taking care of it! :smile:

File: doc/en/announce/release-2.8.2.rst (hunk 1 above)
Comment (The-Compiler) [inline doc/en/announce/release-2.8.2.rst]: `git log --pretty='format:%an' 2.8.1..HEAD | sort | uniq`

Most of those are small doc/test fixes and probably don't warrant a changelog entry, but still nice to give credit here. I probably should add that command to `HOWTORELEASE` :wink:

Comment (nicoddemus) [inline doc/en/announce/release-2.8.2.rst]: Good idea! Thanks! :smile:

**Comment by RonnyPfannschmidt** [comment]
:+1:

**Comment by nicoddemus** [comment]
In case it wasn't clear, :+1: from me as well.  :smile:

**Comment by hackebrot** [comment]
:smile_cat: :+1:

**Pull request merged**